neurom/core/morphology.py

```diff
@@ -240,23 +259,45 @@ def iter_neurites(obj, mapfun=None, filt=None, neurite_order=NeuriteIter.FileOrd
         >>> mapping = lambda n : len(n.points)
         >>> n_points = [n for n in iter_neurites(pop, mapping, filter)]
     """
-    neurites = ((obj,) if isinstance(obj, Neurite) else
-                obj.neurites if hasattr(obj, 'neurites') else obj)
+    def extract_subneurites(neurite):
```

Comment:
can we move this to where it's used

Reply:
Yeah, removed the function and used it directly where it is needed.